task/sbom-json-check/0.1/sbom-json-check.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       name: TEST_OUTPUT
   steps:
   - name: sbom-json-check
-    image: quay.io/redhat-appstudio/hacbs-test:v1.1.0@sha256:82b43bffe4eacc717239424f64478b18f36528df47c2d11df3a8d031e81a3c67
+    image: quay.io/mkosiarc_rhtap/test-hacbs:test2
```

Comment:
I'll change this, waiting for associated PR https://github.com/redhat-appstudio/hacbs-test/pull/117

Reply:
PR merged, waiting for release of a new image